web/src/stories/Button/button.stories.tsx

```diff
@@ -12,7 +14,20 @@ export default {
   decorators: [withDesign, withKnobs],
   parameters: {
     component: Button,
-    docs: { page: docs },
+    docs: {
+      page: docs,
+      container: ({
+        children,
+        context,
+      }: {
+        children: ReactChild | ReactChildren
+        context: Context<Record<string, unknown>>
+      }): JSX.Element => (
+        <DocsContainer context={context}>
+          <ThemeProvider>{children}</ThemeProvider>
```

Comment:
Any ideas about how to provide Theme for all docs files at a single time?

Reply:
We already provide the  `ThemeProvider` within `preview.js` as decorator!

```js 
addDecorator(story => <ThemeProvider>{story()}</ThemeProvider>)
```

You can remove the complete `container` property from the story!